<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ILLCalibration</class>
 <widget class="QWidget" name="ILLCalibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>326</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <widget class="QGroupBox" name="gbInput">
     <property name="title">
      <string>Input</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="MantidQt::MantidWidgets::MWRunFiles" name="rfRunFile" native="true">
        <property name="label" stdset="0">
         <string>Run File</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbCalibGrouping">
     <property name="title">
      <string>Grouping</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QComboBox" name="cbGrouping">
        <item>
         <property name="text">
          <string>Default</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>File</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QStackedWidget" name="swCalibGrouping">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="pgCGDefault"/>
        <widget class="QWidget" name="pgCGMapFile">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="margin">
           <number>0</number>
          </property>
          <item>
           <widget class="MantidQt::MantidWidgets::MWRunFiles" name="rfMapFile" native="true">
            <property name="label" stdset="0">
             <string></string>
            </property>
            <property name="fileExtensions" stdset="0">
             <stringlist>
              <string>.xml</string>
             </stringlist>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbCalibration">
     <property name="title">
      <string>Calibration</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="lbPeakRange">
        <property name="text">
         <string>Peak range:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbScaleFactor">
        <property name="text">
         <string>Scale factor:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QDoubleSpinBox" name="spPeakLower">
        <property name="suffix">
         <string> meV</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>-0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QDoubleSpinBox" name="spPeakUpper">
        <property name="suffix">
         <string> meV</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="minimum">
         <double>-99.989999999999995</double>
        </property>
        <property name="singleStep">
         <double>0.010000000000000</double>
        </property>
        <property name="value">
         <double>0.010000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lbPeakRangeTo">
        <property name="text">
         <string>to</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spScaleFactor">
        <property name="decimals">
         <number>3</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>1.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QCheckBox" name="ckMirrorMode">
        <property name="text">
         <string>Mirror Mode</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QGroupBox" name="gbOutput">
     <property name="title">
      <string>Output</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="ckPlot">
        <property name="toolTip">
         <string>Plot resulting calibration file when created.</string>
        </property>
        <property name="text">
         <string>Plot Result</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_0">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="ckSave">
        <property name="text">
         <string>Save Result</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MantidQt::MantidWidgets::MWRunFiles</class>
   <extends>QWidget</extends>
   <header>MantidQtMantidWidgets/MWRunFiles.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spPeakLower</tabstop>
  <tabstop>spPeakUpper</tabstop>
  <tabstop>spScaleFactor</tabstop>
  <tabstop>ckMirrorMode</tabstop>
  <tabstop>ckPlot</tabstop>
  <tabstop>ckSave</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cbGrouping</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>swCalibGrouping</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
